app/views/admin/configs/show.html.erb

```diff
@@ -464,6 +464,32 @@
             </div>
           </div>
 
+          <div class="card mt-3">
```

Comment:
I just stuck this under `Google Analytics` because alphabetical, but maybe we want to make it a required field in the first Forem setup steps?

Reply:
Actually I think it would be easier to find as a separate section, and also because I think it's more clear in the docs if these sections are separate. We can revisit in the future though!